third_party/move/move-compiler-v2/tests/variable-coalescing/consume_5.move

```diff
@@ -22,9 +22,9 @@ module 0xc0ffee::m {
     public fun test_struct(p: bool, a: W) {
         let b = copy a;
         if (p) {
-            consume_(b);
+            consume_(copy b);
```

Comment:
Not sure why we need this change. Could we please revert it to the original?

If this change is indeed useful, we could add another test with the changes.